### DISCUSSION TITLE
failed to init chain

### DISCUSSION THREAD

**Discussion by denymosh** [NillionNetwork/discussions#130] [Bugs]
when I run nillion-devent, there is error:failed to init chain, tried many ways, still failed. Can you help me?
![1](https://github.com/user-attachments/assets/a4f7f362-1ebb-4080-a2be-65698ab0a1d4)

**Comment by wwwehr** [comment]
Hi @denymosh ! 

Can you provide me:
- exact commands you are using
- your machine details (eg. linux, mac or windows)
- paste me the output of this:
```shell
nilup install latest; nilup use latest; nilup -V; nillion -V
```

**Reply by ScapeXYZ** [reply]
I'm getting same error kindly help with it

**Reply by ScapeXYZ** [reply]
nilup install latest; nilup use latest; nilup -V; nillion -V
SDK version latest already installed
Reinstalling SDK version latest
Installing SDK bins version latest
Downloading latest/nillion-sdk-bins-x86_64-unknown-linux-musl.tar.gz
SDK version latest installed
SDK version latest set as default
nilup 86bb6bd5262c9a223f0895670bba2bfe37ca02ef
tools-config 86bb6bd5262c9a223f0895670bba2bfe37ca02ef

**Reply by ScapeXYZ** [reply]
@wwwehr 

nillion-devnet
`ℹ️ cluster id is 9e68173f-9c23-4acc-ba81-4f079b639964
ℹ️ using 256 bit prime
ℹ️ storing state in /tmp/.tmpFaPnX7 (23.91Gbs available)
🏃 starting nilchain node in: /tmp/.tmpFaPnX7/nillion-chain
Error: failed to init chain

Caused by:
    exited with status: exit status: 1`

**Reply by ScapeXYZ** [reply]
> Hi @denymosh !
> 
> Can you provide me:
> 
> * exact commands you are using
> * your machine details (eg. linux, mac or windows)
> * paste me the output of this:
> 
> ```shell
> nilup install latest; nilup use latest; nilup -V; nillion -V
> ```

I had installed everything in the command 
Still same issues

**Comment by ScapeXYZ** [comment]
How are you able to solve @denymosh that error

**Comment by wwwehr** [comment]
hey @denymosh and @ScapeXYZ 

Can you please re-run with the extra flag:
`nillion-devnet --state-directory /tmp/devnet/`

then please share this file: `/tmp/devnet/nillion-chain/nilchaind.log`

**Comment by ScapeXYZ** [comment]
`nillion-devnet --state-directory /tmp/devnet/`
ℹ️ cluster id is 9e68173f-9c23-4acc-ba81-4f079b639964
ℹ️ using 256 bit prime
ℹ️ storing state in /tmp/devnet/ (<unknown> available)
🏃 starting nilchain node in: /tmp/devnet/nillion-chain
Error: failed to init chain

Caused by:
    exited with status: exit status: 1

**Reply by wwwehr** [reply]
thanks! this is helpful. I'll write back with more shortly

**Reply by ScapeXYZ** [reply]
Do you mean you will get back to me

**Comment by ScapeXYZ** [comment]
`/tmp/devnet/nillion-chain/nilchaind.log`
bash: /tmp/devnet/nillion-chain/nilchaind.log: Permission denied

**Reply by ScapeXYZ** [reply]
Unable to login my nillion devnet localhost because I wasn't able to run Nillion-devnet

**Reply by denymosh** [reply]
I fixed it with another computer, do not know why

**Comment by denymosh** [comment]
I fixed it with another computer, do not know why

**Reply by ScapeXYZ** [reply]
You fixed the error with another computer right

**Reply by ScapeXYZ** [reply]
`nada/src/target/secret_addition.nada.bin`
-bash: nada/src/target/secret_addition.nada.bin: No such file or directory


This is the error I'm facing


Any solution to it

**Reply by ScapeXYZ** [reply]
@wwwehr 
`(.venv) (base) root@DESKTOP-U7PBRHB:~/fun_blind_app/nada# nada/src/target/secret_addition.nada.bin`
-bash: nada/src/target/secret_addition.nada.bin: No such file or directory

**Reply by crypblizz8** [reply]
I believe you solved it @ScapeXYZ ?

**Comment by ScapeXYZ** [comment]
Yes
